### ISSUE TITLE
Deeplink

### ISSUE THREAD

**Issue by saberraz** [lutraconsulting/input#1610]
It would be great to deeplink from external apps/email/messages to a specific coord or item in a specific project

**Comment by saberraz** [comment]
It could be something like:
`app.inputapp.com/open/namespaceA/projectV?proj="xyz"&layer="layerA"&feature="10"`

**Comment by wonder-sk** [comment]
iOS:
- https://medium.com/wolox/ios-deep-linking-url-scheme-vs-universal-links-50abd3802f97
- "universal links" seem like a better option, we would need a dedicated endpoint like https://maps.google.com/apple-app-site-association

Android:
- https://developer.android.com/guide/navigation/navigation-deep-link
- optionally with verification of URI ownership - https://developer.android.com/training/app-links/verify-site-associations - for example https://maps.google.com/.well-known/assetlinks.json

**Comment by timlinux** [comment]
Yeah cool. A good use case would be in OwnTracks I can send a link to the phone when an event happens e.g. entering a geofence area or some other non spatial process (e.g. hourly request to capture data). Then clicking the link in OwnTracks would pop open INPUT in a particular project and start the form capture for a specific feature type.
